forge/ee/routes/teamBroker/schema.js

```diff
@@ -0,0 +1,42 @@
+const YAML = require('yaml')
+module.exports = async function (app) {
+    app.get('/team-broker/schema.yml', async (request, reply) => {
+        const list = await app.teamBroker.getUsedTopics(request.team.hashid)
+        const schema = {
+            asyncapi: '3.0.0',
+            info: {
+                version: '1.0.0',
+                title: `${request.team.name} Team Broker`,
+                description: 'An auto-generated schema of the topics being used on the team broker'
+            }
+        }
+        // Add the team-broker details
+
+        // Figure out the hostname for the team broker
+        let teamBrokerHost = app.config.broker?.teamBroker?.host
+        if (!teamBrokerHost) {
+            // No explict value set, default to broker.${domain}
+            if (app.config.domain) {
+                teamBrokerHost = `broker.${app.config.domain}`
```

Comment:
`domain` might not be set for localfs builds.

I've defaulted to 

```
let defaultHost = app.config.domain ? `broker.${app.config.domain}` : app.config.host
```

if domain isn't set

Reply:
Yeah, I thought I was being clever and added any additional check lower down to only add the `servers` portion to the schema if `teamBrokerHost` had a value... but I realise now, if `domain` isn't set, `teamBrokerHost` *will* have a value of `broker.`... will tidy up.

Reply:
Actually, the logic is sound. `teamBrokerHost` will only get set to a value if either the new setting or `domain` are set. If neither set, it will omit the `servers` section. No changes needed.